tools/validate-login-strings.sh

```diff
@@ -0,0 +1,27 @@
+grep -o '<string name=.*</string>' libs/login/WordPressLoginFlow/src/main/res/values/strings.xml > loginstrings.txt;
+
+declare -a exclusions=('default_web_client_id' 'notification_channel_normal_id' 'notification_channel_important_id')
+
+found_missing_string=false
+string_array=()
+IFS=$'\n'
+while read -r s; do
+  for item in "${exclusions[@]}"; do
+      # Skip lookup if login string is one of the exclusions
+      [[ $s == *$item* ]] && continue 2
+  done
+  s=$(echo $s | sed 's/\\/\\\\/g')
+  if ! grep -oqr "$s" WordPress/src/main/res/values/strings.xml; then
+    string_array+=( $(echo $s | sed 's/\\\\/\\/g') )
+    found_missing_string=true
+  fi;
+done <loginstrings.txt
+
+rm loginstrings.txt
+
+if [ "$found_missing_string" = true ] ; then
+  printf "The following string resources are defined in the login library but are missing or modified in the base app's strings.xml:\n\n"
```

Comment:
The `...or modified in the base apps's strings.xml` might be a bit misleading for someone trying to pinpoint the issue without looking into the script: it could be equally possible that the message is printed out to the console when the modification is done on the library's `strings.xml` file. 
May I suggest we change this to something like `"The following string resources are defined in the login library but are missing in (or differ from) the base app's strings.xml"`?

Reply:
Much better! Updated in fccdb5d.